src/pages/TrueSightV2/components/TokenListVariants.tsx

```diff
@@ -1,4 +1,20 @@
+import { useTheme } from 'styled-components'
+
 import Icon from 'components/Icons/Icon'
+import Row from 'components/Row'
+import { ICON_ID } from 'constants/index'
+
+const VARIANTS: { [key: string]: { icon_id: ICON_ID; title: string } } = {
```

Comment:
duplicate  VARIANTS : src/pages/TrueSightV2/components/SwitchVariantDropdown.tsx

Reply:
they maybe to be modified in future if design changes, should keep them separated for now.